packages/tools/fluid-runner/src/test/sampleCodeLoaders/sample-executable.cjs.ts

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { fluidRunner } from "../../index.js";
```

Comment:
why import differently? why not the below? 
```ts
import { fluidRunner } from "@fluidframework/fluid-runner";
```

Reply:
"@fluidframework/fluid-runner" is the nice way to do it. You could take this example and use it outside of this package.

In the CJS case (surprise to me) the "self-reference" was an error according to VS Code's tsc result. I didn't think that support mattered CJS/ESM and just confirmed it looks like a VS Code limitation/bug. tsc is happy with @fluidframework/fluid-runner. I will update. Thanks for asking. :)

Reply:
Well, I don't know how VS Code is magically right about what should be done. Comment added.